packages/core/src/config/config-resolvers.ts

```diff
@@ -41,25 +41,15 @@ export async function resolveConfig(rawConfig: RawConfig, configPath?: string):
   }
 
   const resolver = new BaseResolver(getResolveConfig(rawConfig.resolve));
-  const configExtends = rawConfig?.styleguide?.extends ?? ['recommended'];
-  const recommendedFallback = !rawConfig?.styleguide?.extends;
-  const styleguideConfig = {
-    ...rawConfig?.styleguide,
-    extends: configExtends,
-    recommendedFallback,
-  };
 
   const apis = await resolveApis({
-    rawConfig: {
-      ...rawConfig,
-      styleguide: styleguideConfig,
-    },
+    rawConfig,
```

Comment:
Please, double check working resolving config with the new config format(flat config)